src/fancy/core/Grid9.hx

```diff
@@ -312,8 +312,11 @@ class Grid9 {
   }
 
   public function sizeRails(topRail: Float, bottomRail: Float, leftRail: Float, rightRail: Float) {
-    if(this.topRail == topRail && this.bottomRail == bottomRail && this.leftRail == leftRail && this.rightRail == rightRail)
-      return;
+    // FIXME: i commented out the early return because sometimes this function
```

Comment:
Agreed ... the split needs to happen. Or move the resizing in refresh ... who know really?